packages/paste-core/components/button-group/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@twilio-paste/button-group",
+  "version": "0.0.0",
+  "category": "interaction",
+  "status": "production",
+  "description": "A Button Group is a component used to render groups of Buttons.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
+    "@twilio-paste/box": "^7.0.1",
+    "@twilio-paste/button": "^11.1.0",
```

Comment:
issue: we still need better tooling for this, but for now you kinda need to manually find all the peerDeps of your peerDeps (recursively) and list them all.  So for example our `button` package depends on 

```
    "@twilio-paste/anchor": "^9.0.0",
    "@twilio-paste/animation-library": "^0.3.1",
    "@twilio-paste/box": "^7.0.0",
    "@twilio-paste/design-tokens": "^8.0.0",
    "@twilio-paste/icons": "^9.0.0",
    "@twilio-paste/spinner": "^11.0.0",
    "@twilio-paste/stack": "^5.0.0",
    "@twilio-paste/style-props": "^6.0.0",
    "@twilio-paste/styling-library": "^1.0.0",
    "@twilio-paste/theme": "^8.0.0",
    "@twilio-paste/types": "^3.1.1",
    "@twilio-paste/uid-library": "^0.2.1",
```

So now `button-group` also needs all those peerDeps.